source/common/router/router.h

```diff
@@ -235,7 +239,39 @@ class FilterConfig {
     for (const auto& upstream_log : config.upstream_log()) {
       upstream_logs_.push_back(AccessLog::AccessLogFactory::fromProto(upstream_log, context));
     }
+    if (config.upstream_http_filters_size() > 0) {
+      auto& server_factory_ctx = context.getServerFactoryContext();
+      const Http::FilterChainUtility::FiltersList& upstream_http_filters =
```

Comment:
the answer is probably no but anything else we can do re code sharing between this and preexisting code?

Reply:
Not without some moderate refactoring. I think there's enough that's sufficiently different that we don't gain much by sharing code.